build/barys

```diff
@@ -396,9 +396,6 @@ else
     # Always activate resinhup bundles in production builds
     sed -i "s/.*RESINHUP ?=.*/RESINHUP ?= \"yes\"/g" conf/local.conf
 fi
-if [ -n "$SUPERVISOR_TAG" ]; then
-    sed -i "s/.*TARGET_TAG ?=.*/TARGET_TAG ?= \"$SUPERVISOR_TAG\"/g" conf/local.conf
-fi
```

Comment:
I can't find where this variable was overwritten either. Could you point us to a code line?